src/components/VaultItemTiles.jsx

```diff
@@ -16,19 +16,21 @@ import { deleteVaultItem } from "@/common/ServerAPI";
 import { deleteItem } from "@/common/stateMutation.jsx";
 
 export default function VaultItemTiles(props) {
-  const { items, setItems } = props
+  const { items, setItems } = props;
```

Comment:
Going along with my comments above, if you always pass an Array into `VaultItemTiles` you could change this prop name from `items` to `itemsArray` and get rid of line 20